db/migrate/20180417154149_create_genre_movies.rb

```diff
@@ -0,0 +1,8 @@
+class CreateGenreMovies < ActiveRecord::Migration[5.1]
+  def change
+    create_table :genre_movies do |t|
+      t.references :movie, foreign_key: true
+      t.references :genre, foreign_key: true
+    end
```

Comment:
thanks for NOT adding timestamps here, it's rare that you WANT timestamps in a join table unless you absolutely need to know WHEN you made that association.